Automation_Custom_Script.sh

```diff
@@ -34,8 +39,12 @@ done
 # Configure passwords and keys
 G_EXEC /boot/dietpi/dietpi-services restart grafana-server
 GLOBAL_PASSWORD=$(openssl enc -d -a -md sha256 -aes-256-cbc -iter 10000 -salt -pass pass:'DietPiRocks!' -in /var/lib/dietpi/dietpi-software/.GLOBAL_PW.bin)
+SERVICE_ACCOUNT_ID=$(\
```

Comment:
Similarly to the api key variable, I'd suggest to prefix this variable with `GRAFANA_`:
```suggestion
GRAFANA_SERVICE_ACCOUNT_ID=$(\
```